README.md

```diff
@@ -204,6 +211,10 @@ Create a `pump.conf` file:
 
 Settings are the same as for the original `tyk.conf` for redis and for mongoDB.
 
+### Tyk Dashboard
+
+The Tyk Dashboard uses the "mongo-pump-aggregate" collection to display analytics.  This is different than the standard "mongo" pump plugin that will store individual analytic items into mongo.  The aggregate functionality was built to be fast, as querying analytics can get expensive when queried in a non-aggregate fashion.
```

Comment:
Maybe clarify that the querying of raw analytics is expensive for large sets

Reply:
@joshblakeley agreed. done.